stl/inc/xthreads.h

```diff
@@ -104,19 +104,21 @@ enum { // mutex types
     _Mtx_recursive = 0x100
 };
 
-_CRTIMP2_PURE _Thrd_result __cdecl _Mtx_init(_Mtx_t*, int);
-_CRTIMP2_PURE void __cdecl _Mtx_destroy(_Mtx_t);
+#ifdef _CRTBLD
+_CRTIMP2_PURE _Thrd_result __cdecl _Mtx_init(_Mtx_t*, int); // used only by _Thrd_create

```

Comment:
I think that `#ifdef _CRTBLD` *means* "used only internally", so such a comment is redundant.

I see your point about bit-rotting, so because I added `#ifdef _CRTBLD`, I think that supersedes the need for comments.

We have traditionally reserved "// TRANSITION, ABI: preserved for binary compatibility" comments for functions that are *completely* unused, except for potential historical callers. The functions in question are currently being used, just STL-internally. Thus I think that TRANSITION comments aren't warranted. We have #3941 tracking the potential for a `_CRTBLD` cleanup.